docs/reference/commandline/tag.md

```diff
@@ -16,5 +16,11 @@ parent = "smn_cli"
 
       --help               Print usage
 
+Image name must match the regular expression `[a-z0-9]+`. 
```

Comment:
Makes sense. In that case, should the command be documented as:

```docker tag [OPTIONS] NAME[:TAG] [REGISTRYHOST/][USERNAME/]NAME[:TAG]```

Note `NAME` instead of `IMAGE`. To me it seems like we want to refer to the name component only.